WordPress/Classes/ViewRelated/Site Creation/Shared/TitleSubtitleTextfieldHeader.swift

```diff
@@ -69,6 +76,10 @@ final class SearchTextField: UITextField {
         autocorrectionType = .no
         adjustsFontForContentSizeCategory = true
 
+        if effectiveUserInterfaceLayoutDirection == .rightToLeft {
+            textAlignment = .right
```

Comment:
This piqued my interest because I initially thought we'd want to go with the `natural` text alignment setting, but that's dependent on the keyboard used, so this makes sense!

Reply:
Thank you Tanner for opening the discussion here 🙇‍♂️

The fact that the `natural` iOS text alignment is based on the keyboard is new to me, I made this change because I was thinking I'm fixing something that doesn't work as expected, but I see it actually works 'natural' if I use a RTL language for the keyboard.

Now it makes me think that I actually looked at the problem from the perspective of a developer fixing an issue for languages he's not using, while in fact for actual users of RTL languages this might feel natural.

TL; DR I didn't have the right context.

Do you think we should keep this change?

Reply:
Your implementation feels right to me based on the default iOS layout for search fields in RTL languages in your description, but perhaps we should consult with a RTL user and get their input to make sure?

Reply:
@yaelirub Pinging you here because you registered the original issue #13607 and maybe you can help!

We'd like to get feedback on this from someone with experience in using RTL languages on an iPhone!

In short: We're not 💯 sure if it's ok to switch the text alignment to right for ⬅️ RTL languages when the keyboard is in a ➡️ LTR language. Given that we're used to always have a ➡️ LTR keyboard it took some time to figure out that the text alignment is naturally switched when the keyboard is in a ⬅️ RTL language.

I tested with a RTL language after our discussion here and it doesn't look bad if this change is removed and everything else stays the same in the pull request.

Please let me know if you checked this comment but you feel you won't have time to look into it this week 🙇

Reply:
👋 Here are the options visually, in case it's helpful:

| Option A | Option B |
| -------- | --------- |
| ![Option A](https://user-images.githubusercontent.com/2092798/155355064-afad0b3f-17b8-4638-aaee-1ab820cc786c.png) | ![Option B](https://user-images.githubusercontent.com/2092798/155355068-63ab1497-94a8-430b-ae1b-9b5432b792bd.png) |

With Option A the text alignment will always match the app localization independently of the keyboard chosen. Option B will align the text for the localization of the individual keyboard (e.g. English will left align, Hebrew will right align).

Reply:
Thanks for the ping @ovitrif . Option B makes more sense to me

Reply:
Thank you Yael 🙇 

@twstokes I removed the part that was forcing the right text alignment regardless of the keyboard.

This is how it behaves now:

| English Keyboard | Hebrew Keyboard |
| --- | --- |
| ![ltr_keyboard](https://user-images.githubusercontent.com/4588074/155759616-8ae7e502-a4ef-48b3-bae2-f5dbb558ee25.png) | ![rtl_keyboard](https://user-images.githubusercontent.com/4588074/155759657-0108f164-2e05-477a-9159-6794908b7e51.png) |

Please give have a look at it when you have time, I'll enable auto-merge to continue if it's all ok 🙇

Reply:
Thanks @ovitrif! I've approved the PR - these screenshots appear how I'd expect them to after recent discussions. 🚀